docs/using-getting-started.asciidoc

```diff
@@ -0,0 +1,304 @@
+// ===============================================================
+
+| event.dataset
+| apache.access
+
+// ==============================================================
+
+| event.kind
+| event
+
+// ===============================================================
+
+| event.category
+| [ "web", "access" ]
+
+// ==============================================================
+
+| event.outcome
+| success
+
+|=====
+
+Lastly, existing field values can be interpreted or enriched using a processor, with the results
+populating additional fields in the final event.
+
+* The `user_agent` processor extracts details from the original user agent string, `user_agent.original`.
+* IP fields like `source.ip` can provide enrichment using the `geopip` processor to add information about the
+  location and autonomous system number (ASN) associated with an IP address.
+* The `registered domain` processor reads a field containing a hostname and writes the registered domain to
+  another field
+* Event collectors, such as https://www.elastic.co/guide/en/beats/libbeat/current/beats-reference.html[Beats], can enrich
+  each event with metadata from the machine's hosting provider (cloud) and/or from the host machine (host).
+
+[options="header"]
+|=====
+| Field Name | Value | Processor
+
+// ==============================================================
+
+| host.architecture
+| x86_64
+| `add_host_metadata`
+
+// ==============================================================
+
+| host.hostname
+| mbp.example.com
+| `add_host_metadata`
+
+// ==============================================================
+
+| host.ip
+| [ "192.168.1.100" ]
+| `add_host_metadata`
+
+// ==============================================================
+
+| host.mac
+| [ "00:00:5E:00:53:23" ]
+| `add_host_metadata`
+
+// ==============================================================
+
+| host.os.build
+| 19E287
+| `add_host_metadata`
+
+// ==============================================================
+
+| host.os.family
+| darwin
+| `add_host_metadata`
+
+// ==============================================================
+
+| host.os.kernel
+| 19.4.0
+| `add_host_metadata`
+
+// ==============================================================
+
+| host.os.name
+| Mac OS X
+| `add_host_metadata`
+
+// ==============================================================
+
+| host.os.platform
+| darwin
+| `add_host_metadata`
+
+// ==============================================================
+
+| host.os.version
+| 10.15.4
+| `add_host_metadata`
+
+// ==============================================================
+
+| user_agent.device.name
+| Other
+| `user_agent`
+
+// ==============================================================
+
+| user_agent.name
+| Chrome
+| `user_agent`
+
+// ==============================================================
+
+| user_agent.os.full
+| Mac OS X 10.15.4
+| `user_agent`
+
+// ==============================================================
+
+| user_agent.os_name
```

Comment:
This should be `user_agent.host.os.name` ;-)

Reply:
Good 👀 thank you! Updating to `user_agent.os.name`